src/ConnectedMode.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -245,14 +247,14 @@ public async Task DownloadQualityProfile_WhenQualityProfileIsNotAvailable_OtherL
             testSubject.InternalState.BindingConfigs.Should().ContainKey(Language.VBNET);
 
             // Progess notifications - percentage complete and messages
-            notifications.AssertProgress(0.25, 0.5, 0.75, 1.0);
+            notifications.AssertProgress(1.0, 2.0, 3.0, 4.0);
```

Comment:
Previously we were using the real progress adapter class which recorded the percentage complete.
Our new dummy implementation just records the number of the step instead i.e. 1 of 4, 2 of 4, etc